pyproject.toml

```diff
@@ -43,10 +43,10 @@ navdf = "nav.bin.navdf:main"
 navdump = "nav.bin.navdump:main"
 naventity = "nav.bin.naventity:main"
 navoidverify = "nav.bin.navoidverify:main"
-navpgdump = "nav.bin.navpgdump:main"
+navpgdump = "nav.pgdump:main"
 navsnmp = "nav.bin.navsnmp:main"
 navstats = "nav.bin.navstats:main"
-navsyncdb = "nav.bin.navsyncdb:main"
+navsyncdb = "nav.pgsync:main"
 navsynctypes = "nav.bin.navsynctypes:main"
```

Comment:
Looking at the naming here I smell something, not sure what. Is it historical that pgdump and pgsync are so different?

Reply:
Historic reason would be that they were both implemented many years after most of the other scripts - and to make them testable (which the legacy scripts were not), their main implementations were placed in the `nav` Python package, rather than directly in the `bin/` directory.  The `bin/` shims were added manually to make them compatible with regular "NAV way" of installing scripts.

An alternative approach to this PR could have been to move `nav.pgsync` and `nav.pgdump` to the `nav.bin` package.